src/sections/header.html

```diff
@@ -1,4 +1,22 @@
-<div>
-    <h1 class="text-blue-800">Header</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
-</div>
\ No newline at end of file
+<section>
+    <nav class="flex justify-between p-4 mx-2 my-6 z-10 relative items-center">
```

Comment:
I'd like to save every utility class for reason. I quite didn't see the need for the `relative`, may be I'm missing something.
```suggestion
    <nav class="flex justify-between p-4 mx-2 my-6 z-10 items-center">
```